### ISSUE TITLE
[Bug]: DC0004 False positive on procedures in internal interface object

### ISSUE THREAD

**Issue by rvanbekkum** [ALCops/Analyzers#438]
### Analyzer

DocumentationCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

DC0004

### Description

When you an `interface` object with `Access = Internal;` then still the DC0004 diagnostic is raised for the procedures in that interface object

### AL code to reproduce

```al
interface SomeObjectName
{
    Access = Internal;

    procedure SomeProcedureName()
}
```

A false positive DC0004 diagnostic is raised on `SomeProcedureName`

**Comment by Arthurvdv** [comment]
I've published a new beta release, `v1.1.0-beta.3`, which should resolve this issue.

Could you verify for me if it's fixed on your end? Would be great to hear whether it's resolved or you're still running into problems.

If you're using the [ALCops VS Code extension](https://marketplace.visualstudio.com/items?itemName=Arthurvdv.alcops), you can switch to the beta channel in the extension settings. To force an update check, run the ALCops: Check for Updates command from the Command Palette.

**Comment by Arthurvdv** [comment]
This should now be resolved with the release of `v1.1.0` of ALCops.

When you have a chance, could you update and verify whether the issue is fixed?

If everything is working as expected, please go ahead and close the issue. If you still run into the problem please leave the issue open and share any additional context, code samples or reproduction steps that might help me investigate further.

**Comment by rvanbekkum** [comment]
I'll check tomorrow and let you know.

**Comment by MODUSCarstenScholling** [comment]
We should reconsider excluding ControlAddin procedures and events. They are an essential part of the contract. While events are excluded from showing the docs (I will raise an issue for AL), they are shown for procedures. This may be very helpful for developers.

<img width="946" height="263" alt="Image" src="https://github.com/user-attachments/assets/c5fe662c-3809-4d02-a82e-3468c6733810" />

**Comment by rvanbekkum** [comment]
> This should now be resolved with the release of `v1.1.0` of ALCops.
> 
> When you have a chance, could you update and verify whether the issue is fixed?
> 
> If everything is working as expected, please go ahead and close the issue. If you still run into the problem please leave the issue open and share any additional context, code samples or reproduction steps that might help me investigate further.

This seems to be resolved!
